### DISCUSSION TITLE
State Management

### DISCUSSION THREAD

**Discussion by AdemAygun** [nylo-core/nylo#37] [Q&A] [answered]
Hi ,
I'm new to flutter.  I've been reviewing nylo framework recently. Nylo is good job. 
It caught my attention that no package related to state management was used. Nylo uses NyState and NyStatefulWidget instead of other state management packages. (In this [link ](https://blog.gskinner.com/archives/2020/09/flutter-state-management-with-mvcs.html), he used MVC+S pattern like nylo with state management )

Do you consider to add a milestone about state management(provider, bloc flutter,or riverpod ) in nylo framework ?

**Comment by agordn52** [comment, marked as answer]
Hi @AdemAygun, 
Thanks for feedback. 

Re. adding a package to handle state management into Nylo.
At the moment there are no plans to do this. Every developer has the freedom to use those packages in the framework (if they want) but we are not favouring one to use.

Out the box, the NyState and NyStateful widget contain all the useful tools you'll need to build an application.
Check the docs to learn more 👍
